heron/scheduler-core/src/java/com/twitter/heron/scheduler/ScalableScheduler.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.twitter.heron.scheduler;
+
+import java.util.Set;
+
+import com.twitter.heron.spi.packing.PackingPlan;
+
+public interface ScalableScheduler {
```

Comment:
The name of "ScalableScheduler" indicates it is a scheduler interface. However, it is for "Scalable" features only. It could be better to remove Scheduler to make it "Scalable" or "IScalable".  Then actual scheduler can : 
class XScheduler implements IScheduler, IScalable {..}